Contracts/categories/v1.0/get-categories/error-response.json5

```diff
@@ -1,21 +1,5 @@
 //Error response object
 {
-	"code": "401",
-	"message": "User is unauthorised",
-	"info":[
-		{
-			"code":"401",
-			"message":"you must login first"
-		}
-	]
-},
-{
-	"code":"404",
-	"message":"Not found",
-	"info":[
-		{
-			"code":"404",
-			"message":"Requested data is not found"
-		}
-	]
+	"status code": "400",
```

Comment:
Error response has to be consistent with all api responses

Reply:
done